vector/src/main/java/im/vector/adapters/VectorMessagesAdapterHelper.java

```diff
@@ -552,6 +552,45 @@ public void onClick(View view) {
         }
     }
 
+    /** Determine how many backticks bracket a message, for code block
+     * handling (issue 145) */
+    public int getTickCount(final Message message) {
+        final String mb = message.body;
+        if (mb.startsWith("```") && mb.endsWith("```")) {
```

Comment:
1 - what is the plan is the message contains several blocks of code ?
you only manage 1 block code by message.

2 - the markdown might be disabled.

Reply:
1. Ah, I didn't know this was a requirement. In that case, if the message contains one or more code blocks, use the adapter_item_vector_message_code.xml  layout, and use spans to set the colour of each block.
2. Will fix this.

Reply:
Issues 1 and 2 both fixed now.